src/monitoring_service/database.py

```diff
@@ -244,6 +244,22 @@ def load_state(self) -> MonitoringServiceState:
         )
         return ms_state
 
+    def channel_close_age(
+        self, token_network_address: TokenNetworkAddress, channel_id: ChannelID
+    ) -> Optional[int]:
+        """ How many blocks ago was the given channel closed? """
+        row = self.conn.execute(
+            """
+                SELECT blockchain.latest_known_block, channel.closing_block
```

Comment:
SQL question: how do I know here that `identifier = ? AND token_network_address = ?` are related to the channel table?

Reply:
Yes, because only that table has columns of that name. But I can make it clearer by prefixing the columns with the table name.